README.md

```diff
@@ -157,6 +157,7 @@ Youtube.authenticate({
 ```
 
 #### Server Key
+*only for request that don't require [user authorization](https://developers.google.com/youtube/v3/guides/authentication) (certain list operations)*
```

Comment:
See [these comments](https://github.com/IonicaBizau/youtube-api/pull/40/files#r38647251)–btw, you can [edit this in this branch](https://github.com/bradoyler/youtube-api/edit/patch-2/README.md) and the pull request will be updated. :four_leaf_clover: